lib/cocina/models/validators/description_values_validator.rb

```diff
@@ -12,29 +12,32 @@ def self.validate(clazz, attributes)
         def initialize(clazz, attributes)
           @clazz = clazz
           @attributes = attributes
-          @error_paths = []
+          @error_paths_multiple = []
+          @error_paths_blank = []
         end
 
         def validate
           return unless meets_preconditions?
 
           validate_obj(attributes, [])
 
-          return if error_paths.empty?
+          # return if error_paths_multiple.empty? && error_paths_blank.empty?
 
-          raise ValidationError, "Multiple value, groupedValue, structuredValue, and parallelValue in description: #{error_paths.join(', ')}"
+          raise ValidationError, "Multiple value, groupedValue, structuredValue, and parallelValue in description: #{error_paths_multiple.join(', ')}" unless error_paths_multiple.empty?
```

Comment:
Should it be able to return both kinds of errors at once?

Reply:
I don't think that's possible, at least as written. The multiples requires the top level `value` to be present and it's not when blank. Perhaps this is a topic for follow up discussion with @andrewjbtw or @arcadiafalcone